falco.yaml

```diff
@@ -480,9 +480,12 @@ plugins:
   - name: json
     library_path: libjson.so
 
+# [Sandbox] `plugins_hostinfo`
+#
 # Uncomment to disable host info support for source plugins
-# that DO NOT generate raw events from the libscap event table,
-# dropping the `hostPath` volume requirement for them.
+# that DO NOT generate raw events from the libscap event table
+# or for plugins that DO NOT parse raw events generated by drivers,
+# effectively dropping the `hostPath` volume requirement for them.
```

Comment:
I think there should be a link to the "hostPath volume requirement".

Because, as a user, reading this, I would search in this file for more information and since there's no other mention of hostPath, would have to start digging through the doc (or maybe even the code) for what that might be.

Reply:
You are right, will put a link to the charts section!

Reply:
Done :)